packages/backend/src/controllers/v2/ParametersController.ts

```diff
@@ -70,8 +70,10 @@ export class ParametersController extends BaseController {
         const results: ApiGetProjectParametersListResults = {
```

Comment:
This mapping seems unnecessary. Doesn't the service method already return `ApiGetProjectParametersListResults`?